docs/Community/CONTRIBUTING.md

```diff
@@ -12,7 +12,7 @@ You'll need the following tools:
 
 - [Git](https://git-scm.com/)
 - [Node.JS](https://nodejs.org/en) >= 20.x
-- [yarn](https://yarnpkg.com/) 4.0.0
+- [yarn](https://yarnpkg.com/) = 4.8.0
```

Comment:
It may be better to change it to `>= 4.0.0` here, because yarn itself is version-locked. As long as it is set to berry, you don't need to pay attention to the specific minor version number.